app/org/sagebionetworks/bridge/models/accounts/User.java

```diff
@@ -30,16 +35,18 @@
     private String email;
     private String healthCode;
     private String studyKey;
-    private boolean consent;
-    private boolean signedMostRecentConsent;
     private SharingScope sharingScope;
     private Set<Roles> roles = Sets.newHashSet();
     private Set<String> dataGroups = Sets.newHashSet();
+    @JsonDeserialize(keyUsing = SubpopulationGuidDeserializer.class)
+    private Map<SubpopulationGuid,ConsentStatus> consentStatuses;
```

Comment:
Serialization test?

Also, where are we serializing this to JSON? This contains the health code and study Id, so we can't expose this to end users.